static/js/reducers/financial_aid.js

```diff
@@ -49,7 +51,7 @@ export const financialAid = (state: FinancialAidState = INITIAL_FINANCIAL_AID_ST
   case START_CALCULATOR_EDIT:
     return { ...FINANCIAL_AID_EDIT, programId: action.payload };
   case CLEAR_CALCULATOR_EDIT:
-    return INITIAL_FINANCIAL_AID_STATE;
+    return _.cloneDeep(FINANCIAL_AID_EDIT);
```

Comment:
actions which are dispatched shouldn't be making direct modifications to the state so it seems like we don't need to clone this here